streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -527,29 +527,26 @@ class StreamingContext private[streaming] (
 
   /**
    * Start the execution of the streams.
-   *
-   * @throws SparkException if the context has already been started or stopped.
```

Comment:
Right. I need to modify it to say SparkException will be thrown only when the context has been already stopped.